src/main/io/gps.c

```diff
@@ -1337,14 +1341,39 @@ static void calculateNavInterval (void)
     gpsSol.navIntervalMs = constrain(navDeltaTimeMs, 50, 2500);
 }
 
+#if defined(USE_VIRTUAL_GPS)
+static void updateVirtualGPS(void)
```

Comment:
(this may be changed to `bool` and return if new sample was generated.  calling code may use it instead of update flag of GPS_MSP)